src/test/java/service/URLCreator.java

```diff
@@ -0,0 +1,24 @@
+package service;
+
+public class URLCreator {
+    private static final String GOOGLE_URL = "testdata.google.url";
+    public static final String WIKIPEDIA_URL = "testdata.wikipedia.url";
```

Comment:
private

Reply:
My apologizes, refactored.
Please see in the following commit.